community/cypher/compatibility-suite/src/main/java/cypher/feature/parser/generated/FeatureResultsBaseListener.java

```diff
@@ -17,7 +17,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-// Generated from /Users/mats/gitRoots/3.0-copy/neo4j/community/cypher/compatibility-suite/src/main/resources/FeatureResults.g4 by ANTLR 4.5.1
+// Generated from /Users/mats/gitRoots/3.0-main/neo4j/community/cypher/compatibility-suite/src/main/resources/FeatureResults.g4 by ANTLR 4.5.1
```

Comment:
Yes, it does suck to have a full path there, since it'll change if you run the code generator on another machine.
The "standard alternative" is of course to not check in the generated files, but in my experience, grammars change rarely enough that the improvements to build time of checking them in and not have to run the code generation step on every build is worth it.